### ISSUE TITLE
?keep-names has no effect

### ISSUE THREAD

**Issue by mootari** [esm-dev/esm.sh#730]
It appears that the `?keep-names` parameter is no longer passed through.

Using the[ example from the docs](https://esm.sh/#esbuild-options), both https://esm.sh/react?keep-names and https://esm.sh/react generate the following response:
```
/* esm.sh - react@18.2.0 */
export * from "https://esm.sh/stable/react@18.2.0/es2022/react.mjs";
export { default } from "https://esm.sh/stable/react@18.2.0/es2022/react.mjs";
```

**Comment by mootari** [comment]
This seems to have been fixed.

**Comment by mootari** [comment]
Appears to have regressed again.

**Comment by ije** [comment]
seems it works for me.

<img width="680" height="99" alt="Image" src="https://github.com/user-attachments/assets/f2882508-9526-4430-8cc6-db0fda69d58c" />

**Comment by mootari** [comment]
Yeah, my last comment was based on the misconception that `keepNames` would apply to all identifiers, not [just functions](https://esbuild.github.io/api/#keep-names). I'll use `dev` instead. Sorry for the noise!
